File: JS/Factorial.js
/*
Factorial 

2! = 1*2
3! = 1*2*3
4! = 1*2*3*4
5! = 1*2*3*4*5
*/


function factorial(n){
    var number = 1
    for(var i=1; i<=n; i++){
        number = number*i;
    }
    return number;
}

console.log(factorial(3))

File: HackerRank/factorial.py
#factorial function, 
'''
5! = 5X4X3X2X1 = 120

'''
import math

def factorial(n):
    ans = 1
    if(n>=1):
        for x in range(1,n+1):
            ans*=x
    return ans



n = int(input())
print(factorial(n))


    

